recipes/cyclus/run_test.sh

```diff
@@ -7,6 +7,14 @@ if [ -z "$CYCLUS_NUC_DATA" ]; then
   export CYCLUS_NUC_DATA="${PREFIX}/share/cyclus/cyclus_nuc_data.h5"
 fi
 
+UNAME="$(uname)"
+if [ "${UNAME}" == "Darwin" ]; then
+  export DYLD_LIBRARY_PATH="${DYLD_LIBRARY_PATH}:${PREFIX}/lib:${PREFIX}/lib/cyclus"
+  export DYLD_FALLBACK_LIBRARY_PATH="${DYLD_FALLBACK_LIBRARY_PATH}:${PREFIX}/lib:${PREFIX}/lib/cyclus"
+else
+  export LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:${PREFIX}/lib/cyclus"
```

Comment:
Looks like there is no `${PREFIX}/lib`. Probably should be `export LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:${PREFIX}/lib:${PREFIX}/lib/cyclus"`.